examples/decomposition/plot_varimax_fa.py

```diff
@@ -69,7 +69,7 @@
     vmax = np.abs(components).max()
     ax.imshow(components, cmap="RdBu_r", vmax=vmax, vmin=-vmax)
     ax.set_yticks(np.arange(len(feature_names)))
-    if ax.is_first_col():
+    if ax.get_subplotspec().is_first_col():
```

Comment:
We only need to modify line 59 and add `share_y=True` and then remove the if/else branch and just call `ax.set_yticklabels(feature_names)`